<!DOCTYPE html>

<html>

<head>

<title>Bo Li's Home Page</title>

<style type="text/css">

#portrait {
    width : 20%;
}

#address {
    float : right;
    margin-right : 60%;
    margin-top : 2%;
}

#clearit {
    clear : right;
}

#research {
    margin-left : 2.5em;
}

</style>

</head>


<body>

<img src="BoLi_portrait.jpg" alt="Bo Li's portrait" id="portrait" />

<div id="address">
<h1>Bo Li</h1>

<p> Postdoctoral Researcher <br />
Center for RNA Systems Biology <br />
University of California at Berkeley </p>

<p> 748 Evans Hall <br />
Berkeley, CA 94720-3840</p>

<p><a href="Curriculum_vitae.pdf"> Curriculum Vitae </a></p>

<p> Email: bli25 [at] berkeley [dot] edu </p>

</div>

<h2 id="clearit"> Academic Employment </h2>

<ul>
<li> 2013.08 &mdash; present &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp Postdoctoral researcher at <a href="http://math.berkeley.edu/~lpachter"> Prof. Lior Pachter </a>'s lab </li>
</ul>

<h2> Education </h2>

<ul>
<li> 2008.09 &mdash; 2013.07 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp Ph.D. in Computer Sciences, &nbsp University of Wisconsin-Madison </li>
<li> 2008.09 &mdash; 2010.05 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp M.S. in Computer Sciences, &nbsp University of Wisconsin-Madison </li>
<li> 2004.09 &mdash; 2008.07 &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp B.E. in Computer Science and Engineering (<strong>ACM Honor Class</strong>), &nbsp Shanghai Jiao Tong University </li>
</ul>

<h2> Research </h2>

<p id="research"> I'm a computational biologist. My research focuses on designing novel algorithms to make sense of next-generation sequencing (NGS) data and tackle bioinformatics challenges in RNA-centric systems biology. I use statistics and machine learning extensively for my research. </p>

<h2> Publications </h2>
<p> * Equal contributors </p>

<ul>
<li><strong> Li, B.</strong>*, Fillmore, N.*, Bai, Y., Collins, M., Thomson, J. A., Stewart, R. and Dewey, C. N. (2014). <a href="http://genomebiology.com/2014/15/12/553/abstract"><strong>Evaluation of <i>de novo</i> transcriptome assemblies from RNA-Seq data</strong></a>. <em>Genome Biology</em>, <strong>15</strong>(12), 553. <a href="http://www.biomedcentral.com/info/about/mostviewed"><img src="highly-accessed.gif" alt="Highly accessed"></a> </li>
<br />
<li>Haas, B., Papanicolaou, A., Yassour, M., Grabherr, M., Blood, P. D., Bowden, J., Couger, M. B., Eccles, D., <strong>Li, B.</strong>, Lieber, M., MacManes, M. D., Ott, M., Orvis, J., Pochet, N., Strozzi, F., Weeks, N., Westerman, R., William, T., Dewey, C. N., Henschel, R., LeDuc, R. D., Friedman, N. and Regev, A. (2013). <a href="http://www.nature.com/nprot/journal/v8/n8/full/nprot.2013.084.html"><strong><i>De novo</i> transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</strong></a>. <em>Nature Protocols</em>, <strong>8</strong>(8), 1494-1512. </li>
<br />
<li> <strong>Li, B.</strong> and Dewey, C. N. (2011). <a href="http://www.biomedcentral.com/1471-2105/12/323"><strong>RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</strong></a>. <em>BMC Bioinformatics</em>. <strong>12</strong>:323. <a href="http://www.biomedcentral.com/info/about/mostviewed"><img src="highly-accessed.gif" alt="Highly accessed"></a> </li>
<br />
<li> Chung, D., Kuan, P. F., <strong>Li, B.</strong>, Sanalkumar, R., Liang, K., Bresnick, E., Dewey, C. N. and Keles, S. (2011). <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002111"><strong>Discovering transcription factor binding sites in highly repetitive regions of genomes with multi-read analysis of ChIP-seq data</strong></a>. <em>PLoS Computational Biology</em>, 7:e1002111. </li>
<br />
<li> <strong>Li, B.</strong>, Ruotti, V., Stewart, R. M., Thomson, J. A. and Dewey, C. N. (2010). <a href="http://bioinformatics.oxfordjournals.org/cgi/content/abstract/26/4/493"><strong>RNA-Seq gene expression estimation with read mapping uncertainty</strong></a>. <em>Bioinformatics</em>, <strong>26</strong>(4), 493-500. </li>
</ul>

<h2> Software </h2>
<ul>
<li><a href="http://deweylab.biostat.wisc.edu/detonate">DETONATE</a>(<i><b>DE</b> novo</i> <b>T</b>ranscript<b>O</b>me r<b>N</b>a-seq <b>A</b>ssembly with or without the <b>T</b>ruth <b>E</b>valuation): A <i>de novo</i> transcriptome assembly evaluation package, which contains two components, RSEM-EVAL and REF-EVAL. I developed the RSEM-EVAL component.</li>
<br />
<li><a href="http://deweylab.biostat.wisc.edu/rsem">RSEM</a>(<b>R</b>NA-<b>S</b>eq by <b>E</b>xpectation-<b>M</b>aximization): Accurate gene and isoform expression levels quantification from RNA-Seq data with or without a reference genome.</li>
<br />
<li><a href="http://deweylab.biostat.wisc.edu/csem">CSEM</a>(<b>C</b>hIP-<b>S</b>eq multi-read allocation using <b>E</b>xpectation-<b>M</b>aximization): A ChIP-Seq multi-read allocator which allows multi-reads to be utilized by peak callers.</li>
<br />
</ul>

<!-- <h2> Miscellaneous </h2> -->

</body>

</html>
